recipes/bugsnag/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "bugsnag" %}
+{% set version = "3.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 67b8c01719e92f193f8424595a94e3a527bc0f9fcb7f2bc47a20af87de81929d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools
```

Comment:
This probably doesn't need pip and setuptools

Reply:
ie you can get rid of setuptools

Reply:
Setup.py uses setuptools though

Reply:
Yep, but setuptools is an implicit dep of pip

Reply:
setuptools dependency removed